compiler/tests/generated/alias_key_types.h

```diff
@@ -7,7 +7,7 @@
 #error This file was generated by a newer version of the Bond compiler and is incompatible with your version of the Bond library.
 #endif
 
-#if BOND_MIN_CODEGEN_VERSION > 0x0b03
```

Comment:
There haven't been any changes to the C++ library or generated code, so we can leave BOND_VERSION at 0x0800.